worker-jobtracking/src/main/java/com/hpe/caf/worker/jobtracking/JobTrackingWorkerReporter.java

```diff
@@ -105,7 +105,7 @@ public JobTrackingWorkerReporter() throws JobReportingException {
     @Override
     public void reportJobTaskProgress(final String jobTaskId, final int estimatedPercentageCompleted) throws JobReportingException
     {
-        LOG.trace("Recieved progress update message for task {}; taking no action", jobTaskId);
+        LOG.trace("Received progress update message for task {}; taking no action", jobTaskId);
```

Comment:
A good change but it shouldn't be in this PR as it has nothing to do with [SCMOD-10455](https://portal.digitalsafe.net/browse/SCMOD-10455).  Changes for different purposes should be in separate PRs (if a PR is required for the change at all).  For an obvious spelling mistake a PR isn't required at all - i.e. this change should go straight into the main branch.

Please review this:
https://github.com/salemove/github-review-helper/blob/master/doc/rules.md

and in particular what it says about keeping commits focused.